react-native-community/react-native-tvos
# Feat/tvos next focus direction

## Summary

I've found android's focus guidance api more powerful, and since it's already there, and possible to implement for iOS (mostly), why not give more options to devs?

## Changelog

[tvOS] [FEAT] - Add directional next focus targetability

## Test Plan

- create 3 views below each other
- target the next one on `nextFocusRight`
- observe the one below getting focus
